src/main/java/seedu/address/model/interview/Rating.java

```diff
@@ -9,7 +9,7 @@
 public class Rating implements Comparable<Rating> {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Rating should only contain a number with one decimal place and between 0.0 to 5.0 inclusive";
+            "Rating should only contain a non-negative number with one decimal place and between 0.0 to 5.0 inclusive";
     public static final String RATING_REGEX = "^([0-4]\\.\\d|5\\.0)$";
```

Comment:
Unrelated to this PR but for the rating, I think it might be possible to use [`Double.parseDouble`](https://docs.oracle.com/javase/8/docs/api/java/lang/Double.html#parseDouble-java.lang.String-) to convert Strings into double values.